### ISSUE TITLE
Skip closed days

### ISSUE THREAD

**Issue by COM8** [TUM-Dev/eat-api#122]
When having a look at the website on the weekend it shows `There is no menu for Sat [...]`.
As a user I'm not interested in this information. So I think we should just skip all those days without menus when the user initially visits the web page.

Also I'd say clicking on "next" and "previous" should skip all days without menus. It should still be possible to enter the date manually to get to a date where there is no menu available.

@Philipp000 would you like to have a look at this?

**Comment by Philipp000** [comment]
I agree, that does not make sense.

However, I think we should indicate more prominently then, for which day the given menu is, so that there are no confusions.

Also, should only weekends be skipped? Or shall we skip to the next day, that is available?
For example, at the moment, there are no menus for February onwards, so I‘m not sure what should happen then 😅

**Comment by COM8** [comment]
For me the date is prominent enough, but in case you have an idea I'm open for it.

I'd say we skip all days without a menu. Not just weekends.
I implemented it by wrapping dates. Here is an example, where I centred the date control. This could perhaps also solve the problem with the date visibility. In case you then click on the date a date selection pops up.
![example date](https://user-images.githubusercontent.com/11741404/137588743-11beb646-fc7d-4ca1-ac51-04f106f89b08.png)
